tutorials/lava/lib/peripherals/dvs/PropheseeCamera.ipynb

```diff
@@ -0,0 +1,313 @@
+    "\n",
+    "# Connect\n",
+    "frame_input.s_out.connect(recv.s_in)\n",
+    "\n",
+    "# Run\n",
+    "num_steps = 200\n",
+    "run_cfg = Loihi2SimCfg()\n",
+    "run_cnd = RunSteps(num_steps=num_steps)\n",
+    "\n",
+    "frame_input.run(condition=run_cnd, run_cfg=run_cfg)\n",
+    "frame_input.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a0a80a0-3d5f-4317-9a95-14b121bbf1d5",
+   "metadata": {},
+   "source": [
+    "## Apply filters\n",
+    "\n",
+    "As you see, the falling balls in the recording cause a trail of events. In many cases these trails are blurring the objects and are undesired. In order to reduce the trails, we can apply the TrailFilterAlgorithm from the metavision_sdk. See the metavision documentation for more filters and their usage."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6a9fc5fb-30d8-4cd4-bb0f-42b2fe0aa5ef",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<table>\n",
+    "<tr>\n",
+    "    <td> <img src=\"gifs/filters.gif\" alt=\"Drawing\" style=\"height: 250px;\"/> </td>\n",
+    "</tr>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "73ef5962-56de-411c-ba02-2113163703d9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filters = [TrailFilterAlgorithm(width=width, height=height, threshold=100000),\n",
+    "           ActivityNoiseFilterAlgorithm(width=width, height=height, threshold=1000),]\n",
+    "\n",
+    "\n",
+    "# Init Processes\n",
+    "frame_input = PropheseeCamera(device=SEQUENCE_FILENAME_RAW,\n",
+    "                              filters=filters,\n",
+    "                              sensor_shape=(height, width),\n",
+    "                              num_output_time_bins=1)\n",
+    "\n",
+    "recv = VisProcess(shape=frame_input.s_out.shape)\n",
+    "\n",
+    "# Connect\n",
+    "frame_input.s_out.connect(recv.s_in)\n",
+    "\n",
+    "# Run\n",
+    "num_steps = 200\n",
+    "run_cfg = Loihi2SimCfg()\n",
+    "run_cnd = RunSteps(num_steps=num_steps)\n",
+    "\n",
+    "frame_input.run(condition=run_cnd, run_cfg=run_cfg)\n",
+    "frame_input.stop()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebade859-eaca-4b07-a6df-3ecf4136df62",
+   "metadata": {},
+   "source": [
+    "## Apply transformations\n",
+    "\n",
+    "We provide numerous transformations to be applied to the event data. For example Downsampling, MergePolarities and Mirroring."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5e91714a-baa6-4dfb-aa62-af0932e017d7",
+   "metadata": {},
+   "source": [
+    "\n",
+    "<table>\n",
+    "<tr>\n",
+    "    <td> <img src=\"gifs/transform.gif\" alt=\"Drawing\" style=\"height: 125px;\"/> </td>\n",
+    "</tr>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "3c87662b-1fe1-4594-bcb0-2a30da9bf41a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transformations = Compose(\n",
```

Comment:
Could you import these transformations in this cell rather than at the beginning of the tutorial? That may clarify that they are part of lava-peripherals rather than metavision.